Userland/Libraries/LibVT/TerminalWidget.cpp

```diff
@@ -243,6 +243,8 @@ void TerminalWidget::keydown_event(GUI::KeyEvent& event)
 
     if (event.key() != Key_Control && event.key() != Key_Alt && event.key() != Key_LeftShift && event.key() != Key_RightShift && event.key() != Key_Super)
         scroll_to_bottom();
+
+    event.ignore();
```

Comment:
This feels too broad, if we did something with the event, we should accept it, not just pass it up the chain.